### PR TITLE
[skip ci] cephadm: add playbook

### PR DESCRIPTION
**Pull request by dsavineau**
This adds a new playbook for deploying ceph via cephadm.

This also adds a new dedicated tox file for CI purpose.

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

File: site-cephadm.yml.sample [hunk 1]
```diff
@@ -0,0 +1,279 @@
+---
```
Comment (guits) [inline site-cephadm.yml.sample:1]: why ".sample" ?

Comment (dsavineau) [reply site-cephadm.yml.sample:1]: like the other site playbooks

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Pull request merged**